website/docs/sdk-reference/react.mdx

```diff
@@ -977,6 +987,48 @@ If you do not want to expose the SDK key or the content of the config JSON file,
 
 Also, we recommend using [confidential targeting comparators](../targeting/targeting-rule/user-condition.mdx#confidential-text-comparators) in the Targeting Rules of those feature flags that are used in the frontend/mobile SDKs.
 
+## Using multiple providers
+
+To access a feature flag's value, your code needs to refer to that Config where that particular feature flag is located. This is what a ConfigCatProvider does for you; it refers to the Config by specifying the Config's sdkKey.
+
+If your code needs access to feature flags from multiple Configs, you will need multiple ConfigCatProviders - each pointing to a Config by specifying that particular Config's sdkKey.
+
+And when your code wants to access a particular feature flag, it should refer to the specific ConfigCatProvider that is associated with the Config where that particular feature flag is located. This is exactly what the providerId allows your code to do: to refer to the specific ConfigCatProvider that is associated with the feature flag it wants to access.
```

Comment:
Nekem Ádámé jobban tetszik, mint a saját javaslatom.

Én magamtól a kötőszavakat írnám csak más sorrendbe, valahogy így.
Ha tetszik bármi az enyémből, akkor szabadon felhasználható.
De ha maarad Ádámé, nekem az is teljesen jó.
Szabin vagyok, légyszi ne várjatok az approve-omra.

`ConfigCatProvider` allows you to access the feature flags of a specific Config, identified by the `sdkKey` parameter. (More precisely, the `sdkKey` identifies a combination of a Config and its Environment.) However, in some cases, you may want to access feature flags from multiple Configs in your application. To do this, you will need to use multiple `ConfigCatProvider`s — each associated with a specific Config by providing the corresponding `sdkKey`.

Keep in mind that `useFeatureFlag` calls will default to using the nearest `ConfigCatProvider`, similar to how React's [useContext](https://react.dev/learn/passing-data-deeply-with-context) works. When working with multiple providers, it may be necessary to differentiate between them. You can achieve this by assigning a unique `id` to each provider (note that the absence of an `id` also acts as a unique identifier).

By passing the appropriate provider `id` in your `useFeatureFlag`, `useConfigCatClient`, or similar calls, you can explicitly specify which provider (i.e., which `sdkKey`, and ultimately which Config and Environment) to use.

Reply:
Köszi a tippeket! 👍 

A jó részét átvezettem a javaslatomba, néhány dolgot leszámítva:

> the `sdkKey`
Amikor kódbeli azonosítóra hivatkozunk, akkor két fajta stílust szoktunk használni: "the `X` parameter" vagy simán `X` névelő nélkül. A "the `X`" nekem nagyon furcsán hat. Ha nem a kódbeli azonosítóról beszélünk, akkor persze semmi gond, viszont akkor írjunk "the SDK Key"-t.

> a combination of a Config and its Environment
A Config és az Environment "merőleges" dolgok, egymástól függetlenül léteznek, az előbbi nem birtokolja az utóbbit. Persze értem, hogy miről van szó, de inkább ne ültessünk ilyen gondolatokat a user fejébe.

> By passing the appropriate
Itt meghagynám a Then-t, mert ez az időbeli sorrendjére utal az említett dolgoknak. Normál esetben először id-kat rendel a providerekhez, aztán hivatkozik ezekre a `useFeatureFlag` hívásokban.